src/components/totals/TotalSupporters/__tests__/TotalSupporters-test.js

```diff
@@ -247,4 +247,28 @@ describe('TotalSupporters', function() {
       expect(icon).not.toBeNull();
     });
   });
+
+  describe('Filtering by page type', function() {
+    var totalSupporters;
+    var element;
+
+    beforeEach(function() {
+      pages.search.mockClear();
+      totalSupporters = <TotalSupporters charityUids={ ["au-24", "au-31"] } type="team" />;
+      element = TestUtils.renderIntoDocument(totalSupporters);
+    });
+
+    it('requests pages only of type "team"', function() {
+      expect(pages.search.mock.calls.length).toEqual(1);
+      expect(pages.search).toBeCalledWith({
+        campaignUid: [],
+        charityUid: ["au-24", "au-31"],
```

Comment:
Not that I'm aware. Though I consider this to be a good thing. Over in line 52, we check that we only use the default props, which subsequently tests for the case where a `type` isn't specified. If these were being tested in the way you described, then that default components test may still pass, even if there was something like `type: null` being passed in. 

Though I definitely don't need to use two charities in this test. That was a copy-paste accident. I'll make them something simpler.